logging/cluster-logging-deploying.adoc

```diff
@@ -43,8 +43,12 @@ include::modules/logging-install-es-operator.adoc[leveloffset=+2]
 [id="cluster-logging-deploying-postinstallation"]
 == Postinstallation tasks
 
-If your network plugin enforces network isolation, xref:#cluster-logging-deploy-multitenant_cluster-logging-deploying[allow network traffic between the projects that contain the {logging} Operators].
+Once you have installed the {clo}, you can configure your deployment by creating and modifying a `ClusterLogging` custom resource (CR).
```

Comment:
Maybe `When` would be better than `Once (one-time only`)?

![Screenshot from 2023-11-28 15-40-21](https://github.com/openshift/openshift-docs/assets/57954076/6d742908-521b-44ed-b761-23a6457c6d61)

Reply:
I think I'll update to "After"?